JourneysSquad/JourneysSquad/View/AddBillView.swift

```diff
@@ -0,0 +1,35 @@
+import SwiftUI
+
+/// View for adding new bill with input fields to enter details of a new bill.
+/// Contain navigation bar with title and button `Back` now.
```

Comment:
nit:

Be consistent and use Present Simple tense:
```suggestion
/// Contains navigation bar with title and button `Back` now.
```

Reply:
Fixed.